lib/bootloader_setup.pm

```diff
@@ -194,12 +194,9 @@ sub boot_local_disk {
         wait_screen_change { send_key 'ret' };
         # Currently the bootloader would bounce back to inst-bootmenu screen after pressing 'ret'
         # on 'local' menu-item, we have to check it and send 'ret' again to make booting properly
```

Comment:
the comment would need to be updated as well then.

Reply:
Yes, the workaround looks can be removed now. I will try to use the openqa-clone-custom-git-refspec to run the branch on o.s.d. since we do not have ppc64le worker locally.

Reply:
In fact, I don't think we need to update the comment since we still check the bonce back once.